src/components/Results/DetectedIssueResources.tsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import { Accordion, AccordionSummary, AccordionDetails, Grid, Typography } from '@material-ui/core';
 import { R4 } from '@ahryman40k/ts-fhir-types';
+import { Enums } from 'fqm-execution';
 import fhirpath from 'fhirpath';
 
 interface Props {
   detectedIssue: R4.IDetectedIssue;
 }
 
 const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
-  const guidanceResponseArray = fhirpath.evaluate(detectedIssue, 'contained.GuidanceResponse');
+  const guidanceResponses = fhirpath.evaluate(detectedIssue, 'contained.GuidanceResponse');
+  const guidanceResponseArray = [];
+
+  for (let i = 0; i < guidanceResponses.length; i++) {
+    if (
+      fhirpath.evaluate(guidanceResponses[i], 'reasonCode.coding.code') === Enums.CareGapReasonCode.MISSING ||
+      fhirpath.evaluate(guidanceResponses[i], 'reasonCode.coding.code') === Enums.CareGapReasonCode.PRESENT
```

Comment:
Let's pull this `fhirpath.evaluate` into a variable before the `if` so we don't have to repeat the same call twice

Reply:
Ah I think I have more info on why I'm seeing my error. This fhirpath expression gives you an array of strings with one element. Not a string itself. So it is actually always failing this check.

indexing to access the first element of the resulting array from this fhirpath evaluate should do the trick

Reply:
Fixed!